File: api/src/main/java/com/ruby/rubia_server/core/enums/ConversationChannel.java
package com.ruby.rubia_server.core.enums;

public enum ConversationChannel {
    WHATSAPP("WhatsApp"),
    WEB("Web"),
    TELEGRAM("Telegram"),
    EMAIL("Email");
    
    private final String displayName;
    
    ConversationChannel(String displayName) {
        this.displayName = displayName;
    }
    
    public String getDisplayName() {
        return displayName;
    }
}

File: api-dotnet/Rubia.Server/Enums/ConversationChannel.cs
using System.ComponentModel;

namespace Rubia.Server.Enums;

public enum ConversationChannel
{
    [Description("WhatsApp")]
    WhatsApp,
    
    [Description("SMS")]
    SMS,
    
    [Description("Email")]
    Email,
    
    [Description("Telegram")]
    Telegram,
    
    [Description("Webchat")]
    Webchat,
    
    [Description("Sistema")]
    System
}